pkg/test/framework/components/echo/common/deployment/echos.go

```diff
@@ -302,18 +302,18 @@ func (c *Config) DefaultEchoConfigs(t resource.Context) []echo.Config {
 		defaultConfigs = append(defaultConfigs, dSvc, eSvc)
 	}
 
-	if !skipDeltaXDS(t) {
-		delta := echo.Config{
-			Service:        DeltaSvc,
+	if !t.Settings().Skip(echo.Sotw) {
```

Comment:
can we remove this check, because i donot find the flag setting skip stow

Reply:
--istio.test.skipWorkloads can skip arbitrary workloads, the old ones like 'skipDelta' are deprecated flags